components/progressbar/index.css

```diff
@@ -127,6 +127,20 @@
 	&.is-negative .spectrum-ProgressBar-fill {
 		background-color: var(--highcontrast-progressbar-fill-color, var(--mod-progressbar-fill-color-negative, var(--spectrum-progressbar-fill-color-negative)));
 	}
+
+	&.spectrum-ProgressBar--staticWhite {
+		&.is-positive .spectrum-ProgressBar-fill {
+			background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
+		}
+
+		&.is-notice .spectrum-ProgressBar-fill {
+			background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
+		}
+
+		&.is-negative .spectrum-ProgressBar-fill {
+			background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
+		}
+	}
```

Comment:
Could you achieve this by increasing the specificity of the existing static white selector on line 240? So the existing styles for static white will override styles for the positive, notice, and negative.
For example, changing 
`.spectrum-ProgressBar--staticWhite {` to 
`.spectrum-ProgressBar.spectrum-ProgressBar--staticWhite {`

Reply:
👏 👏 👏  https://github.com/adobe/spectrum-css/pull/2965/commits/f0e5cb8f3a0ab180e60cb0cb7bcf4d006f8335d9